A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.js

```diff
@@ -1002,5 +1003,209 @@ class A320_Neo_SAI_PressureIndicator extends HTMLElement {
+        this.selfTestSVG.appendChild(st_alt);
+
+        let st_alt_txt = document.createElementNS(Avionics.SVG.NS, "text");
+        st_alt_txt.setAttribute("id", "AltTestTxt")
+        st_alt_txt.textContent = "ALT";
+        st_alt_txt.setAttribute("font", "Roboto");
+        st_alt_txt.setAttribute("font-weight", "900");
+        st_alt_txt.setAttribute("font-size", "42px");
+        st_alt_txt.setAttribute("fill", "#1f242d");
+        st_alt_txt.setAttribute("x", "72.5%");
+        st_alt_txt.setAttribute("y", boxRow2 + txt_off + "%");
+        this.selfTestSVG.appendChild(st_alt_txt);
+
+        let st_tmr = document.createElementNS(Avionics.SVG.NS, "rect");
+        st_tmr.setAttribute("id", "TmrTest")
+        st_tmr.setAttribute("fill", "#afbb3a");
+        st_tmr.setAttribute("x", "30%");
+        st_tmr.setAttribute("y", boxRow3 + "%");
+        st_tmr.setAttribute("width", boxWidthInit + "%");
+        st_tmr.setAttribute("height", boxHeight + "%");
+        this.selfTestSVG.appendChild(st_tmr);
+
+        this.st_tmr_txt = document.createElementNS(Avionics.SVG.NS, "text");
+        this.st_tmr_txt.setAttribute("id", "TmrTestTxt")
+        this.st_tmr_txt.textContent = "INIT "+ Math.ceil(this.testTimer)  +"s";
+        this.st_tmr_txt.setAttribute("font", "Roboto");
+        this.st_tmr_txt.setAttribute("font-weight", "900");
+        this.st_tmr_txt.setAttribute("font-size", "42px");
+        this.st_tmr_txt.setAttribute("fill", "#1f242d");
+        this.st_tmr_txt.setAttribute("x", "31%");
+        this.st_tmr_txt.setAttribute("y", boxRow3 + txt_off + "%");
+        this.selfTestSVG.appendChild(this.st_tmr_txt);
+
+        let st_att = document.createElementNS(Avionics.SVG.NS, "rect");
+        st_att.setAttribute("id", "AttTest")
+        st_att.setAttribute("fill", "#afbb3a");
+        st_att.setAttribute("x", "36%");
+        st_att.setAttribute("y", boxRow1 + "%");
+        st_att.setAttribute("width", boxWidth + "%");
+        st_att.setAttribute("height", boxHeight + "%");
+        this.selfTestSVG.appendChild(st_att);
+
+        let st_att_txt = document.createElementNS(Avionics.SVG.NS, "text");
+        st_att_txt.setAttribute("id", "AttTestTxt")
+        st_att_txt.textContent = "ATT";
+        st_att_txt.setAttribute("font", "Roboto");
+        st_att_txt.setAttribute("font-weight", "900");
+        st_att_txt.setAttribute("font-size", "42px");
+        st_att_txt.setAttribute("fill", "#1f242d");
+        st_att_txt.setAttribute("x", "38%");
+        st_att_txt.setAttribute("y", boxRow1 + txt_off + "%");
+        this.selfTestSVG.appendChild(st_att_txt);
+
+    }
+    update(dTime) {
+        if (!this.complete) {
+            if (this.testTimer >= 0) {
+                this.testTimer -= dTime / 1000;
+            }
+
+            if (this.testTimer > 9) {
+                this.st_tmr_txt.textContent = "INIT " + Math.ceil(this.testTimer) + "s";
+            }
+            else {
```

Comment:
`else` should be on the same line.

Reply:
we dont have a standard for this, as long as its consistent with the code around it